mesa/examples/advanced/sugarscape_g1mt/app.py

```diff
@@ -75,7 +77,11 @@ def portray(g):
 
 page = SolaraViz(
     model,
-    components=[SpaceDrawer, make_plot_component(["Trader", "Price"])],
+    components=[
+        SpaceDrawer,
+        make_plot_component("#Traders"),
```

Comment:
Just curious why "#Traders" and not Just "Traders"

Reply:
I changed the name of the reporter so I had to change it here. For me `#` means "number of".